frontend/src/GameReportForm/index.tsx

```diff
@@ -272,6 +272,14 @@ function GameReportForm({
                         current={formData.competition.value}
                         onChange={handleInputChange("competition")}
                         validate={validateField("competition")}
+                        fakeOptions={
+                            <MultiOptionInputItem
+                                checked
+                                value="Ladder (required)"
```

Comment:
🚲 Do you think we need the "(required)"? Or is that sufficiently implied by the fact that you cannot untick the box? Could it be confusing?

Reply:
On the fence, was thinking if someone tries to untick the box because they don't understand that "rated" == "ladder", and find they can't uncheck the box, they might think it's buggy as a result. Is that a realistic outcome? Or will people get it?

Reply:
moved to discord